tests/test_scalars.py

```diff
@@ -1273,6 +1283,8 @@ def test_timestamp_cmp() -> None:
 
     check(assert_type(ts > c_datetimeindex, np_ndarray_bool), np.ndarray, np.bool_)
     check(assert_type(ts <= c_datetimeindex, np_ndarray_bool), np.ndarray, np.bool_)
+    check(assert_type(ts > c_unknown_index, np_ndarray_bool), np.ndarray, np.bool_)
+    check(assert_type(ts <= c_unknown_index, np_ndarray_bool), np.ndarray, np.bool_)
```

Comment:
Interestingly, these are getting handled by the relational operators in `Index`.

Would be more complete to include them all, i.e., add `<`, `>=`, `==` and `!=`

Reply:
> these are getting handled by the relational operators in Index.

Ah, interesting! I hadn't realised that if an overload couldn't be matched, then the opposite operation from the right-hand-side would be used

updating now, thanks!